R/writeData.R

```diff
@@ -0,0 +1,72 @@
+# Methods for updating Socrata datasets
+###############################################################################
+
+# library('httr')       # for access to the HTTP header
+# library('jsonlite')   # for parsing data types from Socrata
+# library('mime')       # for guessing mime type
+
+#' Wrap httr GET in some diagnostics
+#' 
+#' In case of failure, report error details from Socrata.
+#' 
+#' @param url - Socrata Open Data Application Program Interface (SODA) endpoint (JSON only for now)
+#' @param json_data_to_upload - JSON encoded data to update your SODA endpoint with
+#' @param http_verb - PUT or POST depending on update mode
+#' @param email - email associated with Socrata account (will need write access to dataset)
+#' @param password - password associated with Socrata account (will need write access to dataset)
+#' @param app_token - optional app_token associated with Socrata account
+#' @return httr a response object
+#' @importFrom httr GET
+#' 
```

Comment:
(for consideration) Idea for easier (nicer) debugging would be to add something like ... to the post function